django_app/redbox_app/redbox_core/consumers.py

```diff
@@ -76,19 +77,16 @@ async def receive(self, text_data=None, bytes_data=None):
         user_message_text: str = data.get("message", "")
         selected_file_uuids: Sequence[UUID] = [UUID(u) for u in data.get("selectedFiles", [])]
         user: User = self.scope.get("user")
-        chat_backend = data.get("llm")
         temperature = data.get("temperature")
 
         if session_id := data.get("sessionId"):
             session = await Chat.objects.aget(id=session_id)
-            logger.info("updating: chat_backend=%s -> ai_settings=%s", session.chat_backend, chat_backend)
-            session.chat_backend = chat_backend
+            if llm := data.get("llm"):
```

Comment:
Should we have an info or debug log here to indicate retrieving this from the DB?

Reply:
yes, you are right, i only removed it as i was too lazy to address the async issues!